pyaedt/edb_core/edb_data/components_data.py

```diff
@@ -381,6 +377,11 @@ def refdes(self):
     def refdes(self, name):
         self.edbcomponent.SetName(name)
 
+    @property
+    def is_null(self):
+        """Check if the current object exists."""
```

Comment:
```suggestion
        """Flag indicating if the current object exists."""
```
Suggested change because we want properties described as nouns. If I ever get around to doing a general review of PyAEDT doc, I will be making this change for properties.